kgp/backend/gpml.py

```diff
@@ -101,7 +101,12 @@ def configure(self, input_dim, hyp, opt, inf, mean, cov, lik, dlik,
         """
         self.config = {}
         self.config['lik']  = "{@%s}" % lik
-        self.config['mean'] = "{@%s}" % mean
+
+        if mean_args is None:
+            self.config['mean'] = "{@%s}" % mean
+        else:
+            self.config['mean'] = '{@%s, %s}' % (mean, ', '.join(e if type(e) == str else repr(e) for e in mean_args))
```

Comment:
Do you really need `repr` here? You can simply use `str(e)` instead of the conditional. I suggest:
```python
self.config['mean'] = \
    '{@%s, %s}' % (mean, ', '.join(str(e) for e in mean_args))
```